src/test_workflow/benchmark_test/benchmark_test_cluster.py

```diff
@@ -8,94 +8,46 @@
 
 import json
 import logging
-import os
 import subprocess
-from contextlib import contextmanager
-from typing import Any, Generator, Union
 
 import requests
-import semver
 from requests.auth import HTTPBasicAuth
 from retry.api import retry_call  # type: ignore
 
-from manifests.build_manifest import BuildManifest
-from manifests.bundle_manifest import BundleManifest
 from test_workflow.benchmark_test.benchmark_args import BenchmarkArgs
+from test_workflow.integ_test.utils import get_password
 
 
 class BenchmarkTestCluster:
-    manifest: Union[BundleManifest, BuildManifest]
-    work_dir: str
-    current_workspace: str
     args: BenchmarkArgs
-    output_file: str
-    params: str
-    is_endpoint_public: bool
-    cluster_endpoint: str
     cluster_endpoint_with_port: str
-
-    """
-    Represents a performance test cluster. This class deploys the opensearch bundle with CDK. Supports both single
-    and multi-node clusters
-    """
+    password: str
 
     def __init__(
             self,
-            bundle_manifest: Union[BundleManifest, BuildManifest],
-            config: dict,
-            args: BenchmarkArgs,
-            current_workspace: str
+            args: BenchmarkArgs
+
     ) -> None:
-        self.manifest = bundle_manifest
-        self.current_workspace = current_workspace
         self.args = args
-        self.output_file = "output.json"
-        role = config["Constants"]["Role"]
-        params_dict = self.setup_cdk_params(config)
-        params_list = []
-        for key, value in params_dict.items():
-            if value:
-                '''
-                TODO: To send json input to typescript code from command line it needs to be enclosed in
-                single-quotes, this is a temp fix to achieve that since the quoted string passed from command line in
-                tesh.sh wrapper script gets un-quoted and we need to handle it here.
-                '''
-                if key == 'additionalConfig':
-                    params_list.append(f" -c {key}=\'{value}\'")
-                else:
-                    params_list.append(f" -c {key}={value}")
-        role_params = (
-            f" --require-approval=never --plugin cdk-assume-role-credential-plugin"
-            f" -c assume-role-credentials:writeIamRoleName={role} -c assume-role-credentials:readIamRoleName={role} "
-        )
-        self.params = "".join(params_list) + role_params
-        self.is_endpoint_public = False
-        self.cluster_endpoint = None
```

Comment:
declare `self.cluster_endpoint = args.cluster_endpoint` and then override this class var in BenchmarkCreateCluster class?

Reply:
Sure Rishabh